pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.ruff]
+line-length = 88
+target-version = "py311"
```

Comment:
Can we change this to py312 since that's the version of Python workbenches support.